README.md

```diff
@@ -204,7 +204,7 @@ network.
 To set generate a conf file with permissions set so that only your user can
 read it and write to it:
 
-    (umask 077 && ./cjdroute --genconf > cjdroute.conf)
+    (umask 770 && ./cjdroute --genconf > cjdroute.conf)
```

Comment:
From [Wikipedia](http://en.wikipedia.org/wiki/Umask#Command_line_examples):

umask 077
allow read, write, and execute permission for the file's owner, but prohibit read, write, and execute permission for everyone else

So the readme is correct, no need to fix.